extensions/amp-lightbox-viewer/0.1/service/lightbox-manager-impl.js

```diff
@@ -66,14 +76,19 @@ export class LightboxManager {
     this.lightboxGroups_ = {
       default: [],
     };
+
+    /**
+     * Counter tracking number of carousels without ids
+     * @private {!number}
```

Comment:
nit: primitives are non nullable by default

Reply:
I truly thought I had rid all these files of this thing... welp. Tyty.

Reply:
np, honestly we should have a tool that shouts at us for it. I'm surprised closure doesn't by default